nemo_skills/code_execution/local_sandbox/start_local_sandbox.sh

```diff
@@ -17,7 +17,8 @@
 # NOTE: needs to run from the root of the repo!
 
 SANDBOX_NAME=${1:-'local-sandbox'}
+SANDBOX_MEMORY_LIMIT=${2:-'16g'}
```

Comment:
Why do we need this part?

Reply:
The `MEM_LIMIT_BYTES` in the current container sets memory limits per code execution process. We need to also force a ceiling for the container itself. Without this hard limit I've had my container take down cluster nodes because the container would consume all of the nodes' memory.

Reply:
Actually now that I think about this, the cluster sandbox doesn't go through this script and instead launches the container directly, so you'd need to add this variable to cluster config or set as one of the defaults inside pipeline/utils. So this should have no effect on cluster jobs and not even affect local jobs, only when you directly use this script to start the sandbox. So perhaps it's not needed after all? Or if it is needed, you'd need to pass it differently to the cluster jobs. But perhaps can be done in another pr